e2e/test/scenarios/sharing/public-sharing-embed-button-behavior.cy.spec.js

```diff
@@ -566,8 +562,7 @@ describe("#39152 sharing an unsaved question", () => {
                 titled: true,
                 font: "instance",
                 theme: "light",
-                // TODO: change to expect the correct value of  `downloads`
-                hide_download_button: null,
+                downloads: true,
```

Comment:
From here it's part of a `describeEE("Pro/EE instances",` so it should send the default value (true) unless it's changed later to false